include/maths/CClusterer.h

```diff
@@ -147,7 +147,7 @@ class CClusterer : public CClustererTypes {
     //! \name Clusterer Contract
     //@{
     //! Get the tag name for this clusterer.
-    virtual std::string persistenceTag() const = 0;
+    virtual core::TPersistenceTag persistenceTag() const = 0;
```

Comment:
Is there ever a need for this method to return a dynamically constructed tag rather than a static constant tag?  If not then returning a const reference would avoid the need to copy _two_ strings into the return value.